python/test/unit/language/test_core.py

```diff
@@ -1344,7 +1344,7 @@ def kernel(X, Z, BITCAST: tl.constexpr, SIZE: tl.constexpr):
     if dtype_z.startswith('bfloat'):
         z_tri = torch.empty((size,), dtype=getattr(torch, dtype_z), device=device)
     elif dtype_z.startswith('float8'):
-        z_tri = torch.empty((size,), dtype=torch.float, device=device)
+        z_tri = torch.empty((size,), dtype=torch.half, device=device).to(dtype=getattr(torch, dtype_z))
```

Comment:
does that mean we don't test fp8 to f32 anymore?

Reply:
Shouldn't this test f8->f32: `for dtype_x in torch_float8_dtypes
    for dtype_z in ["float16", "float32", "bfloat16"]`

Reply:
Originally when the destination type is f8, it was getting casted to f32 instead of f8

Reply:
I verified that we still test f8->f32